src/dlgPackageExporter.cpp

```diff
@@ -19,12 +19,30 @@
  *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
  ***************************************************************************/
 
+/* NOTE:
+ *
+ * This should really be called "dlgModuleExporter" - "Packages" should be
+ * reserved for a single node (plus sub-nodes) from one type of Mudlet item that
+ * is "Exported" from the "Editor" whereas "Modules" are collections of possibly
+ * more than one type of Mudlet item and which can include additional reasources
```

Comment:
That is not correct - packages are not reserved for single-node items, they can be multi-node packages just like modules.

Reply:
How, the "Export" option in the Editor only allows one item to be selected IIRC...

Reply:
It's a crappy export option...

Reply:
So? That is how it is isn't it, and it does have its uses to get a straight forward XML file. I guess this might be a bit of a simplistic division between the two terms but actually there is some merit in distinguishing between a _simple_ XML "extract of one leaf/branch of one tree" in the Editor compared to a _more complex_ "collection of multiple branches from multiple trees together with other file based resources all zipped up in an archive file"...

Reply:
I disagree but this isn't the right place to have this discussion, nor is this the i18n PR the right place to sneak remarks into the source code about completely unrelated things :wink:

Reply:
☑️ Reworded to note the difference between what this class does/produces and the "Export" option in the Editor (`dlgTriggerEditor` class) - the latter only exports one "_branch_" of one "_tree_" (widget) of Mudlet items in an `.xml` file whereas this manages a whole "_forest_" (and external files besides) as a `.mpackage` (renamed `.zip`) archive file; but there is no mention of anything about packages vs. modules... 😉